tests/stdlib/unicode/unicode.c3

```diff
@@ -0,0 +1,76 @@
+@require_once "std/format.c3"
```

Comment:
Would be nice to test some error cases/invalid codepoints as well. Make sure we are substituting the maximal subparts correctly.

Reply:
I've added a decoder fuzz test. This doesn't test for maximal subparts but does test the substitutions match the minimum criteria required by the standard.

Reply:
I'm actually very happy with this test.... Its random but so far every subtle incorrectness I've (intentionally) introduced has been caught on every run.

Reply:
I could probably make it smaller without much risk... It only takes 100ms compiled on my machine tho